evm/src/executor/inspector/cheatcodes/mod.rs

```diff
@@ -124,6 +124,9 @@ pub struct Cheatcodes {
     /// Expected revert information
     pub expected_revert: Option<ExpectedRevert>,
 
+    /// Tracks the reverts that happen during evm execution if a there is an expected revert
```

Comment:
let's add a tad more docs on how this works—to document the type properly

Reply:
oky 👌🏼